editor/src/core/model/element-template-utils.ts

```diff
@@ -1120,3 +1121,36 @@ export function elementChildSupportsChildrenAlsoText(
     return null
   }
 }
+
+export type PathPart = Array<string>
```

Comment:
Can just use `ElementPathPart` from `element-path.ts` in place of this without it causing you any grief I believe.

Reply:
Great idea, refactored in [7f7a602](https://github.com/concrete-utopia/utopia/pull/3890/commits/7f7a602d0a4b4f72b115b14ac8dd4e0933b91f22)